src/MCP23017.cpp

```diff
@@ -25,6 +30,17 @@ void MCP23017::init()
 	writeRegister(MCP23017Register::GPPU_A, 0xFF, 0xFF);
 }
 
+void MCP23017::begin()
+{
+	init();
+}
+
+void MCP23017::begin(uint8_t address)
+{
+	_deviceAddr = address;
+	init();

```

Comment:
```suggestion
	_deviceAddr = address;
	begin();
```

In case `begin` would change in the future, it's better to call the other signature than duplicating code. 🙂